src/suit/js/SUIT.js

```diff
@@ -30,25 +31,26 @@ var props = {
     ],
 };
 
+function findCorrectTapService() {
+    const url= window.location.href;
+    const idx= url.indexOf('/portal');
```

Comment:
Can we use `window.location.origin` (might not be supported by older browsers) or `window.location.protocol + '//' + window.location.host` for url or use `widow.location.pathname` instead of `/portal` when finding index? This way it would not depend on current URL.

Reply:
I agree, after thinking about it and looking at @gpdf's comments. I don't think string parsing the best way. I should have  used `URL`. I am going to update it with that.